src/policy/sft_map.rs

```diff
@@ -261,7 +266,7 @@ mod space_map {
 
             let assert_for_index = |i: usize| {
                 let (start, end) = SFTSpaceMap::index_to_space_range(i);
-                debug!("Space: Index#{} = [{}, {})", i, start, end);
+                println!("Space: Index#{} = [{}, {})", i, start, end);
```

Comment:
`println` will print the string to stdout, and will be executed regardless of debug level.  Did you set this to `println` while debugging but forgot to change it back to `debug`?

Reply:
I deliberately left it as `println!()`. Rust logs won't get printed as we do not initialise the logger. This is in the tests, and normally the output will be captured anyway.